### PR TITLE
fixing authority for deployment manager worker and increased default lobby time in game mode

### PR DESCRIPTION
**Pull request by johnprobable**
The deployment manager worker does not have the correct authority set, making it not update the deployment tags properly for the deployment-manager tutorial.

Also, the gamemode only gives about a second to allow players to join a session before the session is made unavailable to new players, and session time is a bit too long.

These changes will fix the authority of the deployment manager worker, change lobby time to half a minute, and session time to 10 minutes.

### CODE REVIEW

File: spatial/one_worker_test.json [hunk 1]
```diff
@@ -1,49 +1,40 @@
...
-      {
-        "name": "streaming_query_diff",
-        "value": "true"
-      },
-      {
-        "name": "bridge_qos_max_timeout",
-        "value": "0"
-      },
+    "dimensions": {
+      "xMeters": 300,
```
Comment (peterimprobable) [inline spatial/one_worker_test.json]: We should keep the dimensions the same as previous config

File: spatial/one_worker_test.json [hunk 2]
```diff
@@ -1,49 +1,40 @@
...
-    "dimensions": {
-      "x_meters": 2000,
-      "z_meters": 2000
+    "chunkEdgeLengthMeters": 50,
+    "snapshots": {
+      "snapshotWritePeriodSeconds": 0
     },
-    "chunk_edge_length_meters": 50,
-    "streaming_query_interval": 4,
-    "legacy_flags": [
```
Comment (peterimprobable) [inline spatial/one_worker_test.json]: Worth checking if any of these legacy flags are required @m-samiec

File: spatial/one_worker_test.json (hunk 1 above)
Comment (johnprobable) [reply spatial/one_worker_test.json]: I'll fix that

File: spatial/one_worker_test.json (hunk 2 above)
Comment (Vatyx) [reply spatial/one_worker_test.json]: The bridge_soft_handover_enabled flag needs to be there I'm pretty sure.
Also how come I don't see the enable_chunk_interest flag?

Comment (johnprobable) [reply spatial/one_worker_test.json]: Will add those two in, though it was definitely working without them. Perhaps they default to false if not present?

Comment (johnprobable) [reply spatial/one_worker_test.json]: Added

**Pull request merged**